src/cluster.c

```diff
@@ -1170,7 +1170,9 @@ RRStatus computeHashSlotOrReplyError(RedisRaftCtx *rr,
             } else {
                 if (*slot != thisslot) {
                     RedisModule_Free(keyindex);
-                    RedisModule_ReplyWithError(ctx, "CROSSSLOT Keys in request don't hash to the same slot");
+                    if (ctx) {
```

Comment:
Probably, it's better to change this function to `computeHashSlot()`. Caller can decide to reply if ctx exists or not.

Reply:
ok done